db/models.py

```diff
@@ -1 +1,22 @@
 from django.db import models
+
+class Race(models.Model):
+    name = models.CharField(max_length=255, unique=True)
+    description = models.TextField(blank=True)
+
+class Skill(models.Model):
+    name = models.CharField(max_length=255, unique=True)
+    bonus = models.CharField(max_length=255)
```

Comment:
The `bonus` field in the `Skill` model is defined as a `CharField`. If this field is intended to store numerical values, consider using an appropriate numerical field type, such as `IntegerField` or `FloatField`.